grails-app/migrations/views/invoice-list.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE VIEW invoice_list AS (
+   	SELECT
+         invoice.id,
+		 invoice.party_from_id,
+         invoice.invoice_number,
+         reference_number.identifier as vendor_invoice_number,
+         unit_of_measure.name as currency,
+         invoice.date_invoiced,
+         person.id as created_by_id,
+         CONCAT(person.first_name, ' ', person.last_name) as created_by_name,
+         count(invoice_item.id) as item_count,
+         count(invoice_item.amount * invoice_item.quantity) as total_value
```

Comment:
I'm not sure what this column is, but I think it should be `sum` instead of `count`

Reply:
yes, it should be sum